### ISSUE TITLE
addGeneExpressionMatrix Error 

### ISSUE THREAD

**Issue by jinoh-neuro** [GreenleafLab/ArchR#1084]
Hi, 

I am trying to add a paired scRNA dataset (in the form of a RangedSummarizedExperiment) from a 10X Multiome Experiment  to my ArchR Project. However, I get the following error:

ArchR logging to : ArchRLogs/ArchR-addGeneExpressionMatrix-14cfc4f9fad94-Date-2021-10-19_Time-17-52-25.log
If there is an issue, please report to github with logFile!
2021-10-19 17:52:27 : Overlap w/ scATAC = 0.981
2021-10-19 17:52:27 : Overlap Per Sample w/ scATAC : atac=5149
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘NSBS’ for signature ‘"CompressedIntegerList"’

Wondering how this error may be fixed. Thank you!

[ArchR-addGeneExpressionMatrix-14cfc4f9fad94-Date-2021-10-19_Time-17-52-25.log](https://github.com/GreenleafLab/ArchR/files/7376885/ArchR-addGeneExpressionMatrix-14cfc4f9fad94-Date-2021-10-19_Time-17-52-25.log)

**Comment by rcorces** [comment]
I dont have enough information to help.

1) Are you able to execute the multiome tutorial without issue? https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html
2) How did you create your seRNA object

**Comment by jinoh-neuro** [comment]
Hi, thanks for the quick response!

To answer your questions:
1. Yes, the multiome tutorial works fine
2. 
seRNA was created by first creating a "SingleCellExperiment" object with DropletUtils read10xCounts function.
sce <- read10xCounts(samples = "path/to/samples", sample.names = names(samples), col.names = TRUE, type = "auto", version = "3", genome = NULL)

I then appended the name to cellNames for matching
colnames(sce) <- paste("sampleName", colnames(sce), sep = "#")

SingleCellExperiment object was then converted to RangedSummarizedExperiment with
seRNA <- as(sce, "RangedSummarizedExperiment")

I had to create the seRNA object in this method, because the .h5 matrix was not available to me, so I was unable to use the import10xFeatureMatrix function
